index.js

```diff
@@ -0,0 +1,175 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let startGameButton = document.createElement("button")
+    let hitButton = document.createElement("button")
+    let stayButton= document.createElement("button")
+    let div1 = document.createElement("div")
+    let div2 = document.createElement("div")
+    let div3 = document.createElement("div")
+    let deck_id = 0
+    let sumOfCards = 0
+    let scores = []
+
+    let title = document.createElement("title")
+    title.innerText = "21 BlackJack"
+    document.head.appendChild(title)
+    
+    const mainPage = () => {
+        let div = document.createElement("div")
+        let h3 = document.createElement("h3")
+        
+        h3.innerText = "Simple 21"
+        startGameButton.innerText = "START GAME"
+        
+        document.body.appendChild(div)
+        div.appendChild(h3)
+        div.appendChild(startGameButton)
+    }
+    
+    const playerPage = async () => {
+        let shuffledDeck = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1")
+        deck_id = shuffledDeck.data.deck_id
+        let drawACard = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/draw/?count=2`)
+        // debugger
+        hitButton.innerText = "HIT"
+        stayButton.innerText = "STAY"
+        div2.className = "playerOneDiv"
+        
+        document.body.appendChild(div1)
+        document.body.appendChild(div2)
+        
+        div1.appendChild(hitButton)
+        div1.appendChild(stayButton)
+
+        let h3 = document.createElement("h3")
+        for(let i = 0; i < 2; i++) {
+            let img = document.createElement("img")
+            let cardValue = drawACard.data.cards[i].value
+            img.src = drawACard.data.cards[i].image
+
+            if(cardValue === "JACK" || cardValue === "QUEEN" || cardValue === "KING"){
+                sumOfCards += 10
+            } else if(cardValue === "ACE") {
+                sumOfCards += 11
+            } else {
+                sumOfCards += Number(drawACard.data.cards[i].value)
```

Comment:
You've already assigned drawACard.data.cards[i].value to a variable cardValue. So you can just use cardValue here.